### ISSUE TITLE
The training set of MFNet is merged from the training and validation set of the original MFNet?

### ISSUE THREAD

**Issue by casiatao** [zifuwan/Sigma#37]
Hello, I downloaded the MFNet from the google drive link you provided. I found that the training set of MFNet is merged from the training and validation set of the original MFNet. Why did you do that? Do the other compared methods in your paper do the same merger?

**Comment by ParkMinyoung0304** [comment]
Hi，by simply changing train.exe, different data in the training set can be called. Compared to directly dividing the training set and validation set images, each has its own advantages and disadvantages

**Comment by casiatao** [comment]
> Hi，by simply changing train.exe, different data in the training set can be called. Compared to directly dividing the training set and validation set images, each has its own advantages and disadvantages

I am asking about how the results in the paper were obtained using what training set, and whether other compared methods in the paper used the same training set or a smaller training set (excluding validation sets), which is essential for fair comparison and correctness of the paper results.

**Comment by ParkMinyoung0304** [comment]
> > Hi，by simply changing train.exe, different data in the training set can be called. Compared to directly dividing the training set and validation set images, each has its own advantages and disadvantages
> 
> I am asking about how the results in the paper were obtained using what training set, and whether other compared methods in the paper used the same training set or a smaller training set (excluding validation sets), which is essential for fair comparison and correctness of the paper results.

I agree with your point. Given that the author's `train.txt` and `test.txt` are predetermined, it’s likely that the dataset used in their comparative experiments is consistent. However, I prefer to randomly shuffle and filter the dataset to minimize the possibility of achieving better performance simply due to the selection of a more favorable dataset.

**Comment by casiatao** [comment]
> > > Hi，by simply changing train.exe, different data in the training set can be called. Compared to directly dividing the training set and validation set images, each has its own advantages and disadvantages
> > 
> > 
> > I am asking about how the results in the paper were obtained using what training set, and whether other compared methods in the paper used the same training set or a smaller training set (excluding validation sets), which is essential for fair comparison and correctness of the paper results.
> 
> I agree with your point. Given that the author's `train.txt` and `test.txt` are predetermined, it’s likely that the dataset used in their comparative experiments is consistent. However, I prefer to randomly shuffle and filter the dataset to minimize the possibility of achieving better performance simply due to the selection of a more favorable dataset.

Thank you for your answer. Are you one of the authors? What I am asking is the details of the author in the experiment. If you are not the author, please do not answer this question before the author answers, thank you！

**Comment by zifuwan** [comment]
Hi, thanks for the questions and discussions. I have uploaded the train.txt file in the google drive and the result in the paper should be reproducible with train.txt. During training the results of Sigma-T will fluctuate between 57.% and 60.2% due to the updated weight (The big fluctuation also exists when we reproduce the baselines, as the MFNet dataset is relatively small). It is recommended to use the training set for fair comparisons.

**Comment by casiatao** [comment]
You still haven't answered whether the other methods used the same train.txt as you did. This is crucial for a fair comparison of their performance. Adding the validation set to the training data will improve performance, but this is not an effect of model improvement.
